packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -189,15 +194,15 @@ void main() {
   );
 
   // Regression test for https://github.com/flutter/flutter/issues/126312.
-  testWidgets('when open input connection in didUpdateWidget, should not throw', (WidgetTester tester) async {
+  testWidgetsWithLeakTracking('when open input connection in didUpdateWidget, should not throw', (WidgetTester tester) async {
     final Key key = GlobalKey();
 
     await tester.pumpWidget(
       MaterialApp(
         home: EditableText(
           key: key,
           backgroundCursorColor: Colors.grey,
-          controller: TextEditingController(text: 'blah blah'),
+          controller: controller,
```

Comment:
This changes the test behavior, switching from testing non empty string to testing empty string. We need to return original behavior.

I suggest to move changes of libs that require more work, like editable_text_test.dart, to separate PR, so that we can merge libs that are ready. This will reduce risk of merge conflicts and mistakes out of it.

Reply:
The issue is still here.